django/thunderstore/community/migrations/0026_schedule_aggregated_fields_refresh.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.7 on 2023-10-18 09:40
+
+import pytz
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    CrontabSchedule = apps.get_model("django_celery_beat", "CrontabSchedule")
+    PeriodicTask = apps.get_model("django_celery_beat", "PeriodicTask")
+
+    schedule, _ = CrontabSchedule.objects.get_or_create(
+        minute="1",
+        hour="*",
+        day_of_week="*",
+        day_of_month="*",
+        month_of_year="*",
+        timezone=pytz.timezone("UTC"),
+    )
```

Comment:
Existing tasks were set to run every 5 and 10 minutes, so they also run on 0 minute. Didn't want to start all the tasks at the same time.

Reply:
Makes sense. It might be a small factor for us too, but I'd say it generally shouldn't matter given that our tasks have a max parallelism already unlike vanilla cronjobs. The reason we have a handful of different task queues is to manage that parallelism and ensure tasks in one queue don't block tasks in others. The way these schedules work is that they just queue a new task, it doesn't actually run immediately.

Nice to see consideration for it anyway 👍